docs/allowedwords.md

```diff
@@ -0,0 +1,21 @@
+## How to add allowed word
+
+All the allowed words are located in an array in [src/config.ts](src/config.ts) inside the object ` ALEX ` as shown below. 
+
+```ts
+ALEX: {
+    profanitySureness: 2,
+    noBinary: true,
+    allow: [
+      'just',
+      'brother-sister',
+      'brothers-sisters',
+      'daft',
+      ...
+    ]
+}
```

Comment:
sorry I think in my previous suggestion the ` ``` ` didn't come through. I think we should close the code block 😃, can you make that change?

Reply:
I pushed a commit that fixes this 👍